addons/mail/static/src/core/common/channel_member_model.js

```diff
@@ -32,10 +32,8 @@ export class ChannelMember extends Record {
         this.id = data.id;
         if ("persona" in data) {
             this.persona = {
-                ...(data.persona.partner ?? data.persona.guest),
-                type: data.persona.guest ? "guest" : "partner",
-                country: data.persona.partner?.country,
-                channelId: data.persona.guest ? data.channel.id : null,
+                ...data.persona,
+                country: data.persona?.country,
```

Comment:
Isn't it a dummy assignation? (the 2nd one) As persona is already spread and the key has the same name.

Reply:
I saw it and planned for follow-up PR, but indeed I can just change the code now